### PR TITLE
Identifier updates 2

### PR DESCRIPTION
**Pull request by 0xABADCAFE**
Focuses on wall drawing code cleanup (work in progress)
- Identifier and label renaming
- Removal of duplicate lookup tables

### CODE REVIEW

File: ab3d2_source/hireswall.s [hunk 1]
```diff
@@ -228,11 +232,11 @@ screendividethru:
 				move.l	(a0)+,d3
 				swap	d3
 				add.l	#DivThreeTable_vb,a5
-				move.w	(a5),StripData
+				move.w	(a5),draw_StripData_w
```
Comment (mheyer32) [inline ab3d2_source/hireswall.s:284]: What denotes the difference between Draw_ and draw_ ?

Comment (0xABADCAFE) [reply ab3d2_source/hireswall.s:284]: As per the proposed style here https://github.com/mheyer32/alienbreed3d2/issues/16 , it's the (assumed) visibility of the identifier. Anything beginning with Draw_ belongs to the drawing code but can be read/written or called from elsewhere. Anything beginning with draw_ belongs to the drawing code but is only used within the drawing code. This helps to indicate the "module" scope of the identifier, given that the identifiers may span several related files belonging in the same conceptual module but are not used outside that scope.

Comment (0xABADCAFE) [reply ab3d2_source/hireswall.s:284]: FYI, Currently there are references in hires.s that clearly belong to other modules, the drawing code, that once all this identification has been carried out can be moved. This is why you'll sometimes see a "private" identifier being used there. I usually examine the context of each use of an identified during the rename before I decide if it's supposed to be public or private.

**Comment by 0xABADCAFE** [comment]
Am I OK to merge?

**Comment by mheyer32** [comment]
Yeah, please go ahead!

**Pull request merged**